0.data_analysis/README.md

```diff
@@ -3,3 +3,4 @@ This module focuses on exploring relationships in the cell morphology data.
 We organize this module into the following analyses:
 
 - `plate_correlation_analyses` explores similarities between aggregated well morphology feature, post feature selection, using genotype and plate groups.
+- `plate_eda_analyses` explores the NF1 plate data.
```

Comment:
Recommend being a bit more specific here. What about the data did you explore?

Reply:
Good point, I'll change this